components/centraldashboard/public/components/header-banner.js

```diff
@@ -0,0 +1,99 @@
+import '@polymer/iron-ajax/iron-ajax.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-item/paper-icon-item.js';
+import '@polymer/paper-ripple/paper-ripple.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-progress/paper-progress.js';
+import './iframe-link.js';
+
+import {html, PolymerElement} from '@polymer/polymer';
+
+import './card-styles.js';
+import '@polymer/paper-styles/element-styles/paper-material-styles.js';
+
+/**
+ * Component to retrieve and display recently modified Jupyter Notebooks.
+ */
+/* eslint-disable */
+export class HeaderBanner extends PolymerElement {
+    static get template() {
+        return html`
```

Comment:
JUST A COMMENT NOT A CHANGE REQUEST : Thank you for doing this component in just html and avoiding pug.